packages/lib/src/createUniqueClassName.ts

```diff
@@ -0,0 +1,25 @@
+// https://stackoverflow.com/questions/105034/how-to-create-a-guid-uuid
```

Comment:
`crypto.randomUUID` couldn't be used because it was released with node v15.6.0 unfortunately. 

I think this would be fine from a performance perspective since it runs in constant time? Let me know if you have specific lines you're concerned about.

From a legibility perspective, I guess the bitwise operators can be tricky for some, but it's a pretty elegant and performant way to handle this operation. I feel like if there's any point of confusion, we should handle it with comments instead? I figured it would be fine since I left the SO link in and the thread has a pretty good explanation of how this works.

Reply:
My main concern is that some applications may be executing this 40-50 times once the page loads (multiplied by potentially several JS bundles that at least one application has). More a prompt than a direct concern.

My note about legibility was primarily just around renaming variables to improve legibility, not necessarily rewriting the code to not use bitwise operators.

Reply:
Could you clarify specifically which line you see as the performance concern, or if there was a specific alternative on the thread you saw as more performant? I'm not sure what change to make based on that comment.

The function already runs in constant time, so the only possible improvements I see are in the regions of replacing `performance.now()` with `Date.now()`, which doesn't make that much of a difference.

Below is a quick test of how quickly this function runs as it is currently.
<img width="275" alt="image" src="https://user-images.githubusercontent.com/99368875/159300992-433957c9-47ae-4f41-ac8a-3b94f2847f1c.png">

I'm pretty doubtful this will cause performance issues? Again, if I'm missing something, just let me know and I'll make the change.

Reply:
Often, it's better to lean on built-ins from both a performance and maintenance perspective. As you noted, the crypto library isn't supported widely enough yet, so sounds like that's not a viable option here. Just double-checking before we go and add this to a bunch of components!